consent/strategy_default.go

```diff
@@ -939,6 +935,7 @@ func (s *DefaultStrategy) issueLogoutVerifier(ctx context.Context, w http.Respon
 		return nil, errorsx.WithStack(fosite.ErrInvalidRequest.
 			WithHint("Logout failed because none of the listed audiences is a registered OAuth 2.0 Client."))
 	}
+	cl.Secret = "" // We don't want to expose the client secret.
```

Comment:
Was this a bug prior? Or is this just a safe guard?

Reply:
I see, that makes sense. Client should instead really implement a custom JSON marshaller that removes the secret + a type alias which doesn't. But I see that this is not the case and we have a bunch of `c.Secret=""` everywhere.